model-engine/model_engine_server/api/llms_v1.py

```diff
@@ -356,6 +356,11 @@ async def create_completion_sync_task(
             detail=f"Upstream service error for request_id {request_id}",
         )
     except (ObjectNotFoundException, ObjectNotAuthorizedException) as exc:
+        if isinstance(exc, ObjectNotAuthorizedException):  # pragma: no cover
+            logger.info(
+                f"POST /completion_sync to endpoint {model_endpoint_name} for {auth} failed with authz error {exc.args}"
```

Comment:
nit: the route is `/completions-sync`

Reply:
oh lol, all the other log messages use this path. but will updated